src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -372,10 +371,28 @@ public IEnumerable<SubscriptionGeneralDataAndStats> GetAllSubscriptions(Transact
             }
         }
 
+        public static IEnumerable<SubscriptionState> GetAllSubscriptionsWithoutState(TransactionOperationContext serverStoreContext, string database, int start, int take)
```

Comment:
DevelopmentHelper.ToDo - move to cluster state machine

Reply:
moved